node/test/PublicAPITest.ts

```diff
@@ -1,11 +1,22 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2021 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
 import { assert } from 'chai';
 import * as SignalClient from '../index';
 
+SignalClient.initLogger(
+  SignalClient.LogLevel.Trace,
+  (level, target, fileOrNull, lineOrNull, message) => {
+    const targetPrefix = target ? '[' + target + '] ' : '';
+    const file = fileOrNull ?? '<unknown>';
+    const line = lineOrNull ?? 0;
+    // eslint-disable-next-line no-console
+    console.log(targetPrefix + file + ':' + line + ': ' + message);
```

Comment:
To work smoothly within our system, you'd want to map your error level to our levels. In the renderer process, we have `window.log.warn`, `window.log.error`, and `window.log.info`.

Note that this will not work if you are loaded in our main process. In that world, `console.log/warn/error` are available.

We could potentially override `console.warn/error` in the renderer process as well. To make this easier.

Reply:
This is the local test; we'll be able to call `initLogger` however it makes sense for the actual app. I'm hesitant to have the main package set it up automatically because of load order, though.